.github/workflows/RFC-2119-usage.yml

```diff
@@ -0,0 +1,16 @@
+name: "Check if PR introduces errant RFC-2119"
+on: 
+  pull_request:
+      types: [opened, edited, assigned, closed, review_requested, ready_for_review]
+      paths:
+        - index.html
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check PR
+      uses: JJ/github-pr-contains-action@releases/v10
+      with:
+        github-token: ${{github.token}}
+        diffDoesNotContain: " must | must not | required | shall | shall not | should | should not | recommended | may | optional "
```

Comment:
remove "required", "optional", "recommended" and make follow up issue to discuss removing these from the spec.

Reply:
✅  here is the issue to track this: https://github.com/w3c/aria/issues/1916